http/api_handler.go

```diff
@@ -259,6 +259,7 @@ var apiLinks = map[string]interface{}{
 	"tasks":     "/api/v2/tasks",
 	"checks":    "/api/v2/checks",
 	"telegrafs": "/api/v2/telegrafs",
+	"plugins":   "/api/v2/telegraf/plugins",
```

Comment:
part of me thinks this is its own resource `/telegraf-plugins`, its a mouth full, but the nested `/telegraf/plugins` is odd in REST, especially when we have the `telegrafs` as the resource. 🤔

Reply:
I did it that way to keep from needing an `/api/v2/` route for everything telegraf related; plugins for now, possibly bundles in the future, etc... By keeping telegraf related things under the `/telegraf` resource, we avoid potential future conflicts with features influxdb might add. the current `/telegrafs` can't serve anything generic yet related to telegraf because the `/telegrafs/:id` bit allows anything.
I'm not sure if it's "proper" as it's been a minute since i've designed api's, but i would have done something like `query` for telegraf's bit:
```go
"telegraf": map[string]string{
	"plugins": "/api/v2/telegraf/plugins",
	"configs": "/api/v2/telegraf/configs", # replace current `/api/v2/telegraf`
}
```
***But***, if it's "the restful way", i definitely agree this should be it's own resource, whether `plugins` or `telegraf-plugins`